src/solver.cpp

```diff
@@ -700,6 +701,17 @@ inline SolverRegistrar::SolverRegistrar(IdRegistryData& registry)
              Primitive::MultiMarginLoss,
              multimarginloss::MultiMarginLossForward{}.SolverDbId());
 
+    Register(registry,
+             ++id,
+             Primitive::SparseSoftmaxCrossEntropyWithLogits,
+             sparsesoftmaxcrossentropywithlogits::SparseSoftmaxCrossEntropyWithLogitsForward{}
+                 .SolverDbId());
+    Register(registry,
+             ++id,
+             Primitive::SparseSoftmaxCrossEntropyWithLogits,
+             sparsesoftmaxcrossentropywithlogits::SparseSoftmaxCrossEntropyWithLogitsBackward{}
+                 .SolverDbId());
+
```

Comment:
dont leave a new line between the newly added op and the below comment, refer to the following [comment](https://github.com/ROCm/MIOpen/blob/dfb40febe6d7d0def0decfe8c658cff076ec607a/src/solver.cpp#L712-L713)

Reply:
resolved